sunkit_pyvista/plotter.py

```diff
@@ -72,6 +74,25 @@ def set_camera_coordinate(self, coord):
         pos = tuple(camera_position[0])
         self.plotter.camera.position = pos
 
+    def set_view_angle(self, angle: u.deg = 30):
+        """
+        Sets the view angle of the camera by the specified value
+        in degrees.
+
+        Parameters
+        ----------
+        angle : `astropy.units.Quantity`
+            The viewing angle.
+        """
+        view_angle = angle.to_value()
+        if not view_angle > 0 and view_angle <= 180:
+            raise ValueError("specified view angle must be "
+                             "0 deg < angle <= 180 deg")
+        # Default zoom = 1 where view angle = 30 degrees
+        # Zoom/view_angle = 1/30
+        zoom_value = 30 / view_angle
```

Comment:
I think calling `.zoom()` below will zoom relative to the currently view_angle, so this should be 
```suggestion
        zoom_value = self.camera.view_angle / view_angle
```

Reply:
Ah yes, this is correct. Sorry